### PR TITLE
TriggerOptions is added to postNewTrigger for server code.

### PR DESCRIPTION
**Pull request by dairyo**
TriggerOptionsをserver code用のpostNewTriggerに追加しました。

Related PR: #167

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/ThingIFAPI+Trigger.swift [hunk 1]
```diff
@@ -86,13 +88,30 @@ extension ThingIFAPI {
         
         // generate body
         let requestBodyDict = NSMutableDictionary(dictionary: ["predicate": predicate.toNSDictionary(), "serverCode": serverCode.toNSDictionary(), "triggersWhat": TriggersWhat.SERVER_CODE.rawValue])
+        if let title = triggerOptions.title {
+            requestBodyDict["title"] = title
+        }
+        if let description = triggerOptions.triggerDescription {
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/ThingIFAPI+Trigger.swift]: please use short statements, like 

``` swift
requestBodyDict["title"] = options?.title
```

same for description and metadata

Comment (dairyo) [reply ThingIFSDK/ThingIFSDK/ThingIFAPI+Trigger.swift]: Thanks! I will fix it.

**Comment by dairyo** [comment]
@yongpingchen 

I fixed according to your comment.
I also fixed postNewTrigger for command. It has same issue.

**Comment by yongpingchen** [comment]
LGMT

**Pull request merged**